scripts/west_commands/create_board/ncs_create_board.py

```diff
@@ -172,8 +172,8 @@ def do_run(self, args, unknown_args):
         with open(out_dir / f"Kconfig.{input['board']}", "w") as f:
             f.write(tmpl.render())
 
-        tmpl = env.get_template("board.yml.jinja2")
-        with open(out_dir / f"board.yml", "w") as f:
+        tmpl = env.get_template("board.yaml.jinja2")
+        with open(out_dir / f"board.yaml", "w") as f:
```

Comment:
no board uses `board.yaml` and it is not supported, `yaml` is a glob for twister, `yml` was used because twister won't look at it